client/my-sites/upgrades/domain-management/dns/dns-add-new.jsx

```diff
@@ -166,7 +169,7 @@ const DnsAddNew = React.createClass( {
 
 				<FormFooter>
 					<FormButton
-						disabled={ formState.isSubmitButtonDisabled( this.state.fields ) || this.props.isSubmittingForm }
+						disabled={ this.state.show ? isSubmitDisabled : false }
```

Comment:
Definitely :+1: Maybe at around the same time we introduce editing the records :wink: (I'm half-serious here, that might actually be a good time for that)